datamover/pkg/drivers/lifecycle/expiration.go

```diff
@@ -39,7 +39,7 @@ func deleteObj(objKey string, lastmodifed int64, virtBucket string, bkend *Backe
 	}
 
 	// delete metadata
-	delMetaReq := osdss3.DeleteObjectInput{Bucket: virtBucket, Key: objKey, Lastmodified: lastmodifed}
+	delMetaReq := osdss3.DeleteObjectInput{Bucket: virtBucket, Key: objKey/*, Lastmodified: lastmodifed*/}
```

Comment:
why is lastmodifed commented ?

Reply:
@rhsakarpos thanks for your review, since object delete function is not finished yet, can we fix this problem in next pr ? thanks.